src/app/shared/models/classes/product-attribute.class.ts

```diff
@@ -0,0 +1,11 @@
+import { ProductAttributeOption } from "@shared/models/types/product-attribute-option.interface"
+
+export class ProductAttribute {
+    name: string;
+    options: ProductAttributeOption[];
+
+    constructor(name: string, options: ProductAttributeOption[]) {
```

Comment:
please use the same structure as other classes (using ProductAttributeShape) so that we can init the class using an object instead of a list of parameters